bin/start.js

```diff
@@ -29,6 +29,12 @@ const start = async (userConfig, packageDir, logFile) => {
     logFile,
   );
 
+  let debug = true;
+
+  if ('wp' in userConfig && 'debug' in userConfig.wp) {
```

Comment:
In this case `userConfig` isn't the full `cypress.json` configuration but just the `wp` key. Therefore we only need to do something like:

` const debug = userConfig.debug ? true : false;`